com.unity.multiplayer.mlapi/Runtime/Logging/NetworkLog.cs

```diff
@@ -67,8 +66,14 @@ private static void LogServer(string message, LogType logType)
                 {
                     using (var nonNullContext = (InternalCommandContext) context)
                     {
-                        nonNullContext.NetworkWriter.WriteByte((byte) logType);
-                        nonNullContext.NetworkWriter.WriteStringPacked(message);
+                        var bufferSizeCapture = new CommandContextSizeCapture(nonNullContext);
+                        using (bufferSizeCapture.Measure())
```

Comment:
Since this is a class this will allocate each time we enter this and create a lot of gc when the profiler is attached. If we are fine with that please ignore. Just wanted to raise it in case we were not aware.

Reply:
Whoops, this was meant to be a struct. Good catch!